File: C++/9A-Die Roll.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    int a, b, prob, maximum;

    cin >> a >> b;

    maximum = max(a, b);

    if (maximum == 1)
    {
        cout << "1/1" << endl;
    }
    else if (maximum == 2)
    {
        cout << "5/6" << endl;
    }
    else if (maximum == 3)
    {
        cout << "2/3" << endl;
    }
    else if (maximum == 4)
    {
        cout << "1/2" << endl;
    }
    else if (maximum == 5)
    {
        cout << "1/3" << endl;
    }
    else
    {
        cout << "1/6" << endl;
    }

    return 0;
}

File: python/9A - Die Roll.py
a,b = map(int, input().split())

maximum = max(a,b)

if maximum == 1:
    print('1/1')
elif maximum == 2:
    print('5/6')
elif maximum == 3:
    print('2/3')
elif maximum == 4:
    print('1/2')
elif maximum == 5:
    print('1/3')
else:
    print('1/6')